### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by dprothero**
This fixes a number of issues found with this sample. See the individual commit messages for details on what was fixed.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
+const express = require("express");
+const bodyParser = require("body-parser");
+const path = require("path");
 const app = express();
-const tokenGenerator = require('./token_generator');
+const tokenGenerator = require("./token_generator");
 
 // load configuration from .env if available
-require('dotenv').config();
+require("dotenv").config();
```
Comment (bld010) [inline server.js:8]: Do we still need this if you took out the `process.env.PORT`?

Comment (dprothero) [reply server.js:8]: Yes, it pulls in the API Keys, service SIDs, etc. from the .env file.

Comment (bld010) [reply server.js:8]: The config.js file also calls it, though?

Comment (dprothero) [reply server.js:8]: Oh, I see... we're calling it twice. Good catch. I'll push up a fix.

**Review by onlywade — APPROVED** [review]
Thanks for these improvements!

**Pull request merged**